compiler/crates/relay-typegen/src/write.rs

```diff
@@ -178,7 +178,10 @@ pub(crate) fn write_operation_type_exports_section(
     write_import_actor_change_point(actor_change_status, writer)?;
     runtime_imports.write_runtime_imports(writer)?;
     write_fragment_imports(typegen_context, None, encountered_fragments, writer)?;
-    write_relay_resolver_imports(imported_resolvers, writer)?;
+    // TODO: Add proper support for Resolver type generation in typescript
```

Comment:
This is follow-up work for us, right? Can you link a WI/issue?

Reply:
#4772 Added